eradiate/xarray/util.py

```diff
@@ -0,0 +1,9 @@
+from eradiate import unit_registry as ureg
+
+
+def to_quantity(variable):
+    try:
+        units = variable.units
+    except KeyError:
+        raise ValueError(f"{variable} has no units.")
+    return ureg.Quantity(variable.values, units)
```

Comment:
Stuffing many specific helper functions without well-though API integration is opening the door to lots of dead code in the long term. Since this function is tiny and you seem to use it in a single module, I'd rather have it there as a local `_to_quantity()` function.

Reply:
Alright!